sqlmesh/core/snapshot/evaluator.py

```diff
@@ -381,15 +381,23 @@ def _create_snapshot(self, snapshot: Snapshot, snapshots: t.Dict[SnapshotId, Sna
             )
         else:
             logger.info("Creating table '%s'", table_name)
-            self.adapter.create_table(
-                table_name,
-                query_or_columns_to_types=snapshot.model.columns_to_types
-                if snapshot.model.annotated
-                else snapshot.model.ctas_query(parent_snapshots_by_name, is_dev=is_dev),
-                storage_format=snapshot.model.storage_format,
-                partitioned_by=snapshot.model.partitioned_by,
-                partition_interval_unit=snapshot.model.interval_unit(),
-            )
+            if snapshot.model.annotated:
```

Comment:
why did this change?

Reply:
As I was refactoring a few things I found the dual purpose of `create_table` to be confusing. I prefer being intentional on if I am creating an empty table vs. creating a table with data (CTAS).